### PR TITLE
Removed server distaria, add server tariamc (Distaria was renamed TariaMC)

### PR DESCRIPTION
**Pull request by EinsLucaaa**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

## Further comments

Distaria was renamed TariaMC

### CODE REVIEW

File: minecraft_servers/tariamc/manifest.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        "teamspeak": "tariamc.net"
+    },
+    "brand": {
+        "text": "#FFFF",
+        "background": "#306fdb",
+        "primary": "#306fdb"
+    },
+    "discord": {
+        "server_id": 952950096843120690,
+        "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/tariamc/manifest.json:39]: Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.

File: minecraft_servers/tariamc/manifest.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            "name": "Boxxed",
+            "color": "#3471eb"
+        },
+        "starblock": {
+            "name": "StarBlock",
+            "color": "#FFE81F"
+        }
+    },
+    "social": {
+        "web": "https://tariamc.net/",
```
Comment (JakobBruening) [inline minecraft_servers/tariamc/manifest.json:27]: Remove "/" at the end.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

**Comment by EinsLucaaa** [comment]
Done

**Review by JakobBruening — APPROVED** [review]
Thank you! 💪🏽

**Pull request merged**